ScriptEngine/API/EntityAPI.cpp

```diff
@@ -31,11 +31,18 @@ ClassDefine<EntityClass> EntityClassBuilder =
         .instanceProperty("blockPos", &EntityClass::getBlockPos)
         .instanceProperty("maxHealth", &EntityClass::getMaxHealth)
         .instanceProperty("health", &EntityClass::getHealth)
+        .instanceProperty("canFly", &EntityClass::getCanFly)
```

Comment:
remove `get` in `getCanXxx` or `getIsXxx` plz, this looks strange

Reply:
> remove `get` in `getCanXxx` or `getIsXxx` plz, this looks strange

**Changed!**